Docs/source/running_cpp/profiling.rst

```diff
@@ -1,46 +1,35 @@
 Profiling the code
 ==================
 
-Profiling with AMREX's built-in profiling tools
------------------------------------------------
-See `this page <https://amrex-codes.github.io/amrex/docs_html/Chapter12.html>`__ in the AMReX documentation.
-
-
-Profiling the code with Intel Advisor on NERSC
-----------------------------------------------
-
-Follow these steps:
-
-- Instrument the code during compilation
+Profiling allows us to find the bottle-necks of the code as it is currently implemented.
+Bottle-necks are the parts of the code that may delay the simulation, making it more computationally expensive.
+Once found, we can update the related code sections and improve its efficiency.
+Profiling tools can also be used to check how load balanced the simulation is, i.e. if the work is well distributed accross all MPI ranks used.
+Load balancing can be activated in WarpX by setting the input parameter `warpx.load_balance_int <https://warpx.readthedocs.io/en/latest/running_cpp/parameters.html#distribution-across-mpi-ranks-and-parallelization>`__.
```

Comment:
A bit simpler and more robust to use links to rst files.
```suggestion
Load balancing can be activated in WarpX by setting input parameters, see the parallelization section at :doc:`parameters`.
```

Reply:
Thanks @MaxThevenet ! That looks much better, yes.